spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt

```diff
@@ -0,0 +1,803 @@
+    onPrimary: Color = if (isPro) Blue20 else Orange20,
+    primaryContainer: Color = if (isPro) Blue30 else Orange30,
+    onPrimaryContainer: Color = if (isPro) Blue90 else Orange90,
+    secondary: Color = Grey99,
+    onSecondary: Color = Color.Black,
+    secondaryContainer: Color = Grey30,
+    onSecondaryContainer: Color = Grey90,
+    tertiary: Color = GreyBlue40,
+    onTertiary: Color = GreyBlue90,
+    tertiaryContainer: Color = GreyBlue20,
+    onTertiaryContainer: Color = GreyBlue95,
+    error: Color = Red80,
+    onError: Color = Red20,
+    errorContainer: Color = Red30,
+    onErrorContainer: Color = Red90,
+    valid: Color = Green80,
+    onValid: Color = Green20,
+    validContainer: Color = Green30,
+    onValidContainer: Color = Green90,
+    background: Color = Grey10,
+    onBackground: Color = Grey90,
+    surface: Color = Grey10,
+    onSurface: Color = Grey80,
+    surfaceVariant: Color = GreyBlue30,
+    onSurfaceVariant: Color = GreyBlue80,
+    surfaceTint: Color = primary,
+    outline: Color = GreyBlue50,
+    outlineVariant: Color = GreyBlue30,
+    inversePrimary: Color = Orange40,
+    inverseSurface: Color = onSurface,
+    inverseOnSurface: Color = Grey20,
+    scrim: Color = Grey0,
+) = SparkColors(
+    primary = primary,
+    onPrimary = onPrimary,
+    primaryContainer = primaryContainer,
+    onPrimaryContainer = onPrimaryContainer,
+    secondary = secondary,
+    onSecondary = onSecondary,
+    secondaryContainer = secondaryContainer,
+    onSecondaryContainer = onSecondaryContainer,
+    tertiary = tertiary,
+    onTertiary = onTertiary,
+    tertiaryContainer = tertiaryContainer,
+    onTertiaryContainer = onTertiaryContainer,
+    error = error,
+    onError = onError,
+    errorContainer = errorContainer,
+    onErrorContainer = onErrorContainer,
+    valid = valid,
+    onValid = onValid,
+    validContainer = validContainer,
+    onValidContainer = onValidContainer,
+    background = background,
+    onBackground = onBackground,
+    surface = surface,
+    onSurface = onSurface,
+    surfaceVariant = surfaceVariant,
+    onSurfaceVariant = onSurfaceVariant,
+    surfaceTint = surfaceTint,
+    outline = outline,
+    outlineVariant = outlineVariant,
+    scrim = scrim,
+    inversePrimary = inversePrimary,
+    inverseSurface = inverseSurface,
+    inverseOnSurface = inverseOnSurface,
+)
+
+
+/**
+ * A color scheme holds all the named color parameters for a [SparkTheme] highly inspired by Material 3 ColorScheme.
+ *
+ * SparkColors are designed to be harmonious, ensure accessible text, and distinguish UI
+ * elements and surfaces from one another. There are two built-in baseline schemes,
+ * [lightSparkColors] and a [darkSparkColors], that can be used as-is or customized.
+ *
+ * To learn more about colors, see [Material Design colors](https://m3.material.io/styles/color/overview) and
+ * [Spark colors](https://zeroheight.com/25c15666f/p/40105d-colors/b/77b3e7).
```

Comment:
Do we want to link to Brikke?

Reply:
for now I think so since they're not ready yet